lib/backend.js

```diff
@@ -287,6 +287,35 @@ exports.createServer = (dynamodb, docClient) => {
     })
   )
 
+  app.delete('/api/deleteAllTables', asyncMiddleware(async (req, res) => {
+    try {
```

Comment:
We don't need to explicitly use the try/catch block. `asyncMiddleware` takes care of that.